src/System.Text.Encodings.Web/tests/EncoderExtensionsTests.cs

```diff
@@ -2,7 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.Framework.WebEncoders;
+using System.Text.Encodings.Web.Tests;
```

Comment:
Ideally this would be within the `S.T.Encodings.Web.Tests` namespace as well and we wouldn't need this using, but given we have the custom adapter encoders, it makes sense to keep this as is for now (and remove the temporary adapters in the future).